apps/src/applab/applab.js

```diff
@@ -847,15 +847,38 @@ Applab.init = function(config) {
   }
 };
 
-Applab.appendToEditor = function(newCode) {
+/**
+ * @param {string} newCode Code to append to the end of the editor
+ */
+Applab.appendToEditor = function (newCode) {
   var code = studioApp.editor.addEmptyLine(studioApp.editor.getValue()) + newCode;
   studioApp.editor.setValue(code);
 };
 
+Applab.scrollToEnd = function () {
```

Comment:
Was wondering that myself. Some of this code does seem pretty droplety. What do you think?